data/json/recipes/recipe_others.json

```diff
@@ -7904,5 +7904,36 @@
     [ [ "wheel", 1 ] ],
     [ [ "motor", 1 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "neoprene",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_OTHER",
+  "skill_used": "cooking",
+  "skills_required": [ "cooking", 4 ],
+  "difficulty": 4,
```

Comment:
Ill raise it to 5 then. Then mid-game books can at least provide it while still being more difficult than a 4. If 5 is unacceptable as well, ill raise it to 6. But that'll really reduce the usefulness of the material.